src/forms/utils/ohri-form-helper.ts

```diff
@@ -13,3 +17,28 @@ export function cascadeVisibityToChildFields(
       field.isParentHidden = visibility;
     });
 }
+
+export function inferInitialValueFromDefaultFieldValue(
+  field: OHRIFormField,
+  context: EncounterContext,
+  handler?: SubmissionHandler,
+) {
+  if (!isEmpty(field.questionOptions.defaultValue)) {
+    if (field.questionOptions.rendering == 'toggle') {
+      if (field.questionOptions.defaultValue == ConceptTrue) {
```

Comment:
Remove this `if` please. Just do `field.questionOptions.defaultValue == ConceptTrue`

Reply:
Also what is `ConceptTrue`? I don't see it being passed as a parameter neither being created or assigned on this method. Where does this variable comes from

Reply:
I understand now what is `ConceptTrue`, I think this code should be part of the validate method... you can't have the `infer..` method handling this kind of stuff.

Reply:
> Remove this if please. Just do field.questionOptions.defaultValue == ConceptTrue

True that, thanks, updated.

The validator just validates whether the provided value is valid depending on the `rendering` type of a question. The `inferInitialValueFromDefaultFieldValue` function is a helper function that does the heavy lifting, it's job is to:

1. Check whether a field has an initial value (This has been changed, the invoker of the function should handle that)
2. Check if value is valid
3. Invoke the underlying submission handler to generate an `obs` to be submitted.